src/assessor_tools.yml

```diff
@@ -84,27 +106,6 @@
         install_dir: /tools/DotNetToJScript
         unarchive_extra_opts:
           - --strip-components=1
-    - role: assessment_tool
-      vars:
-        archive_src: "https://github.com/ChrisTruncer/Egress-Assess/\
-        tarball/master"
-        install_dir: /tools/Egress-Assess
-        pip_packages:
-          - pyftpdlib
-        powershell: yes
-        python2: yes
-        unarchive_extra_opts:
-          - --strip-components=1
-    - role: assessment_tool
-      vars:
-        archive_src: "https://github.com/BC-SECURITY/Empire/tarball/master"
-        csharp: yes
-        install_dir: /tools/Empire
-        pip_packages:
-          - .
-        powershell: yes
-        unarchive_extra_opts:
-          - --strip-components=1
```

Comment:
That's certainly one way to deal with it!

Reply:
Sometimes you just have to trim that fat man lol